app/controllers/users_controller.rb

```diff
@@ -7,6 +7,9 @@ def index
   end
 
   def show
+    @publications = Publication.joins(:participants).where(participants: { user_id: params[:id] })
```

Comment:
Let's add the associations in the model and use that for publications (and activities). See relationship between Activities/Publications and news articles News Articles, that is done throw ContentNews. And let me know if you need help.

Setting that up would allow us to do: `@user.publications`